### ISSUE TITLE
ORG_FULL_NAME argument to org-create is parsed wrong

### ISSUE THREAD

**Issue by jblaine** [chef/chef-server#354]
Chef Server 12.0.3

sudo /opt/opscode/bin/chef-server-ctl org-create --help usage says:

```
knife opc org create ORG_SHORT_NAME ORG_FULL_NAME (options)
```

This command:

```
$ sudo /opt/opscode/bin/chef-server-ctl org-create r701-dmz "R701 DMZ" -a jblaine -f /tmp/r701-dmz-validation.pem
```

Results in an ORG_FULL_NAME of just "R701" not "R701 DMZ"

**Comment by yoshiwaan** [comment]
Just encountered this myself, so :+1:

**Comment by stevendanna** [comment]
Thanks for filing this bug report. This was fixed in Chef 12.0.4 with commit 27700dbf9d65e29421fae6818ea5cb7e96f30532
